gadget-container/gadgets/bcck8s/bcc-wrapper.sh

```diff
@@ -134,7 +134,9 @@ if [ "$MANAGER" = "true" ] ; then
   MODE="--mntnsmap"
   MAPPATH=$BPFDIR/gadget/mntnsset-$TRACERID
   CGROUP_V2_PATH=$(cat /proc/self/cgroup |grep ^0:|cut -d: -f3)
-  if [ ! -z "$CGROUP_V2_PATH" ] && [ "$CGROUP_V2_PATH" != "/" ]; then
+  UID_UNDER=`echo $GADGET_CONTAINER_UID | sed 's/-/_/g'`
+  if [[ "$CGROUP_V2_PATH" == *"$GADGET_CONTAINER_UID"* ||
+      "$CGROUP_V2_PATH" == *"$UID_UNDER"* ]]; then
```

Comment:
This assumes that if the Gadget pod uses cgroupv2, then other pods too. I wonder if that's still true when the Gadget daemonset is deployed in a different [runtime class](https://kubernetes.io/docs/concepts/containers/runtime-class/) than other pods, and the runtime classes are configured differently with regards to cgroups...

But I guess we can ignore this corner case for now.

Reply:
I agree to ignore this for now, it seems nasty to fix.